contactmps/models.py

```diff
@@ -167,6 +167,7 @@ class Email(models.Model):
     # This makes no promises about keys being present from one email to another.
     any_data = JSONField()
     campaign = models.ForeignKey(Campaign, related_name='emails')
+    moderate = models.BooleanField(default=1, help_text="Moderate the submission")
```

Comment:
I recommend you rather call this needs_moderation - right now I can't look at the name and the value and know what it means - I have to look at the condition where it's used.

Reply:
This is also gonna affect all campaigns - we have other campaigns online that still offer people the chance to send messages even though we won't be moderating them - I think we need to be able to select the moderation functionality per campaign.

Reply:
An alternative would be to disable the existing campaigns and make it clear to users they're now closed (on their templates) - e.g. they're still included in news24 somewhere. Then we just start with the assumption that all campaigns need moderation from now on until we change that.